.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Enhancement
+about: Help improve performance, usability and/or consistency.
+label: maintenance
+---
+
+### Describe the issue
+
+**Summary**
+
+<!-- A short 1-2 sentences that succinctly describe what could be improved -->
+
+**Code example**
+<!--A minimal code snippet required to reproduce the bug.
+Please make sure to minimize the number of dependencies required, and provide
+any necessary plotted data
+-->
+
```

Comment:
This isn't really a bug fix, this is a tweak/optimization/probably the why would we want to do this?